src/scripts/routes.js

```diff
@@ -0,0 +1,440 @@
+define([
+    "emby-button",
+    "emby-input",
+    "scripts/livetvcomponents",
+    "paper-icon-button-light",
+    "emby-itemscontainer",
+    "emby-collapse",
+    "emby-select",
+    "livetvcss",
+    "emby-checkbox",
+    "emby-slider",
+    "listViewStyle",
+    "dashboardcss"], function () {
+
+    function defineRoute(newRoute, dictionary) {
+        var baseRoute = Emby.Page.baseUrl();
+        var path = newRoute.path;
+        path = path.replace(baseRoute, "");
+        console.log("Defining route: " + path);
```

Comment:
hmm, why?..

Reply:
Well i just moved from site.js to this file. But i made a commit optimizing the function.